.github/ISSUE_TEMPLATE/content-review-request.md

```
@@ -0,0 +1,68 @@
+        - General Reveiw
+      default: 0
+  - type: checkboxes
+    id: checkboxes-2
+    attributes:
+      label: "Post-Review Followup? "
+      description: Do you want the reviewer to reach out to you following the review
+        process?
+      options:
+        - Yes - I want to know the outcome
+        - No - I do not need to know the outcome
+  - type: input
```

markdownlint-cli2 (0.17.2)

Unordered list indentation
Expected: 0; Actual: 2

(MD007, ul-indent)